### PR TITLE
Align to scans and fix Du Bois' author page

### PR DESCRIPTION
**Pull request by brendanny**
- that blockquote isn't actually a letter (and isn't a blockquote in the scans)

- added css to dedication

**Edit**: - fix link to Du Bois's author page

(and I really need to figure out my commits before i force push... hope you don't have GH notis on)

### CODE REVIEW

File: src/epub/text/passing.xhtml [hunk 1]
```diff
@@ -50,9 +50,7 @@
...
-					</blockquote>
+					<p>“… For I am lonely, so lonely⁠ ⁠… cannot help longing to be with you again, as I have never longed for anything before; and I have wanted many things in my life.⁠ ⁠… You can’t know how in this pale life of mine I am all the time seeing the bright pictures of that other that I once thought I was glad to be free of.⁠ ⁠… It’s like an ache, a pain that never ceases.⁠ ⁠…” Sheets upon thin sheets of it. And ending finally with, “and it’s your fault, ’Rene dear. At least partly. For I wouldn’t now, perhaps, have this terrible, this wild desire if I hadn’t seen you that time in Chicago.⁠ ⁠…”</p>
```
Comment (acabal) [inline src/epub/text/passing.xhtml]: We do want this to be a letter though, it clearly says hse's reading a letter in the previous paragraph.

**Review by acabal — CHANGES_REQUESTED** [review]

File: src/epub/text/passing.xhtml (hunk 1 above)
Comment (brendanny) [reply src/epub/text/passing.xhtml]: her dialog is mixed in with non-dialog in that paragraph though

Comment (brendanny) [reply src/epub/text/passing.xhtml]: could split it into two blockquotes?

Comment (acabal) [reply src/epub/text/passing.xhtml]: OK, then we should end the blockquote where the quote ends, start a p, then a new blockquote to wrap up the last sentence.

**Comment by brendanny** [comment]
OK, this should be good

**Comment by acabal** [comment]
Thanks!

**Pull request merged**